CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ different versioning scheme, following the Haskell community's
 * C# NuGet version: TBD (minor bump needed)
 * C# Comm NuGet version: TBD (minor bump needed)
 
+### C++ & `gbc` ###
+
+* Remove FieldTemplate details from field reflection type.
```

Comment:
Minor: can you expand on this and say why we're making the change (to shorten the symbol names)?